Scripts/create_deb_packages.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# create binary *.deb packages
+
+#set -x # DEBUG
+
+mkdir -p build
+cd build
+cmake -DRDK_INSTALL_INTREE=OFF -DCMAKE_INSTALL_PREFIX=/usr ../
```

Comment:
This creates a pretty stripped-down RDKit build (without things like InChI ior AvalonTools ntegration), wouldn't it make sense to include those?